src/NotesLoaderBMS.cpp

```diff
@@ -1487,7 +1494,7 @@ void BMSSongLoader::AddToSong()
 		for( unsigned i = 0; i < loadedSteps.size(); i ++ )
 		{
 			Steps *steps = loadedSteps[i].steps;
-			steps->SetDifficulty( Difficulty_Medium );
+			//steps->SetDifficulty( Difficulty_Medium );
```

Comment:
As before - BMSes are found in order and sorted by difficulty rating - only the lowest difficulty used is now "Easy" (was previously Medium). So three BMSes in a folder without difficulty tags will be given easy, medium and hard difficulties.